cmd/agent/cmd/daemon.go

```diff
@@ -24,18 +24,17 @@ func SetupUtility() {
 func DaemonMain() {
 	rootLogger.Sugar().Infof("config: %+v", types.AgentConfig)
 
-	SetupHttpServer()
-	initGrpcServer()
+	SetupHealthHttpServer()
+	SetupAppHttpServer()
 	// TODO: udp server, tcp server, websocket server
-
 	if types.AgentConfig.AppMode {
 		// app mode, just used to debug
 		rootLogger.Info("run in app mode")
 		// sleep forever
 		select {}
 	} else {
 		rootLogger.Info("run in agent mode")
-
+		initGrpcServer()
```

Comment:
initGrpcServer also could be test in app mode

Reply:
done